keyboards/gmmk/pro/pro.h

```diff
@@ -48,3 +48,32 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     { k90, k91, k92, k93, k94, k95, ___, k97}, \
     { ___, ka1, ka2, ka3, ka4, ka5, ka6, ka7}  \
 }
+
+//  ESC  F1   F2   F3   F4   F5   F6   F7   F8   F9   F10  F11  F12  Prt    Rotary(Mute)
+//  ~    1    2    3    4    5    6    7    8    9    0    -    (=)  BSpc     Del
+//  Tab  Q    W    E    R    T    Y    U    I    O    P    [    ]    Enter    PgUp
+//  Caps A    S    D    F    G    H    J    K    L    ;    "    #             PgDn
+//  Sh_L /    Z    X    C    V    B    N    M    ,    .    ?  Sh_R       Up   End
+//  Ct_L    Win_L   Alt_L      SPACE          Alt_R   FN   Ct_R     Left Down Right
+
+#define LAYOUT_iso( \
+    k13, k26, k36, k31, k33, k07, k63, k71, k76, ka6, ka7, ka3, ka5, k97,     k01, \
+    k16, k17, k27, k37, k47, k46, k56, k57, k67, k77, k87, k86, k66, ka1,     k65, \
+    k11, k10, k20, k30, k40, k41, k51, k50, k60, k70, k80, k81, k61, ka4,     k15, \
+    k21, k12, k22, k32, k42, k43, k53, k52, k62, k72, k82, k83, k84,          k25, \
```

Comment:
```suggestion
    k11, k10, k20, k30, k40, k41, k51, k50, k60, k70, k80, k81, k61,          k15, \
    k21, k12, k22, k32, k42, k43, k53, k52, k62, k72, k82, k83, k84, ka4,     k25, \
```
ISO Enter should be on the next line by convention:
https://github.com/qmk/qmk_firmware/tree/master/layouts/default